app/models/manageiq/providers/amazon/storage_manager/s3/cloud_object_store_container/operations.rb

```diff
@@ -0,0 +1,15 @@
+module ManageIQ::Providers::Amazon::CloudManager::CloudObjectStoreContainer::Operations
+  extend ActiveSupport::Concern
+
+  def raw_destroy
+    raise "CloudObjectStoreContainer has no #{ui_lookup(:table => "ext_management_systems")}, unable to destroy it" unless ext_management_system
+    with_provider_object(&:delete!)
+    destroy
```

Comment:
I am saying the caller to `raw_destroy` should be checking policy before calling this provider level method.  That way the policy check is performed for all providers from one spot.